kiwi/bootloader/config/grub2.py

```diff
@@ -827,20 +823,16 @@ def _setup_bios_image(self, mbrid=None, lookup_path=None):
             ['bash', '-c', bash_command]
         )
 
-    def _create_embedded_fat_efi_image(self):
-        Path.create(self.boot_dir + '/boot/' + self.arch)
-        efi_fat_image = ''.join(
-            [self.boot_dir + '/boot/', self.arch, '/efi']
-        )
+    def create_efi_loader_image(self, path):
```

Comment:
I like your changes and this makes sense to me. Thanks for starting the refactor.

Regarding the proposed change of the function name I would not do it for the following reasons:

* by convention a method starting with "_" is a private method for which we don't provide a stable API. The new name indicates this is now a public method of the bootloader API. I don't think we should do this
* the new name is misleading to me. ```create_efi_loader_image``` indicates you are creating an EFI image. This is not what the method does. The former name ```_create_embedded_fat_efi_image``` describes what it actually does. When booting ISO images in EFI mode the ISO has to provide a fat filesystem because the EFI standard requires this. The method here wraps an existing EFI image into the standard required FAT filesystem structure. It does not create an EFI image.

So if you don't mind, can we keep the name of the method ?

Reply:
> by convention a method starting with "_" is a private method for which we don't provide a stable API. The new name indicates this is now a public method of the bootloader API. I don't think we should do this

I thought `_` was to indicate private methods which are only called from inside the class itself (and child classes). I'll change it.

> So if you don't mind, can we keep the name of the method ?

Will do.